app/conf/assets.conf

```diff
@@ -303,5 +306,9 @@ loadSets = {
 	
 	3dmodels = [
 		mrdoob/three, mrdoob/detector, mrdoob/STLLoader, mrdoob/PLYLoader, mrdoob/CTMLoaderLZMA, mrdoob/CTMLoaderCTM, mrdoob/CTMLoader, mrdoob/trackballcontrols
-	],	
+	],
+
+	queryBuilder = [
+	    jquery/jquery, jquery/moment, jquery/queryBuilder, jquery/queryBuilderCss, ca/querytranslator
```

Comment:
Yep good point.  And I should remove `moment` because it's already included in `queryBuilder` (which is actually a standalone version).